File: src/wsTripXML/Code/OTAClasses/OTAMultiMessageRS.vb
Option Strict Off
Option Explicit On 

Imports System.Xml.Serialization

Namespace wsTravelTalk.wmMultiMessageOut

    '<remarks/>
    <System.Xml.Serialization.XmlRootAttribute(IsNullable:=False)> _
    Public Class MultiMessageRS

        '<remarks/>
        Public Success As Success

        '<remarks/>
        <System.Xml.Serialization.XmlArrayItemAttribute(IsNullable:=False)> _
        Public Errors() As [Error]

        '<remarks/>
        Public Response As String

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute()> _
        Public Version As String
    End Class

    '<remarks/>
    <System.Xml.Serialization.XmlRootAttribute(IsNullable:=False)> _
    Public Class Success
    End Class

    '<remarks/>
    <System.Xml.Serialization.XmlRootAttribute(IsNullable:=False)> _
    Public Class [Error]

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute()> _
        Public Type As String

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute()> _
        Public Code As String

        '<remarks/>
        <System.Xml.Serialization.XmlTextAttribute()> _
        Public Value As String
    End Class

    '<remarks/>
    <System.Xml.Serialization.XmlRootAttribute(IsNullable:=False)> _
    Public Class Errors

        '<remarks/>
        <System.Xml.Serialization.XmlElementAttribute("Error")> _
        Public [Error]() As [Error]
    End Class
End Namespace

File: src/wsTripXML/Code/OTAClasses/OTAMultiMessageRS.cs

using System.Xml.Serialization;

namespace wsTripXML.wsTravelTalk.wmMultiMessageOut
{

    // <remarks/>
    [XmlRoot(IsNullable = false)]
    public class MultiMessageRS
    {

        // <remarks/>
        public Success Success;

        // <remarks/>
        [XmlArrayItem(IsNullable = false)]
        public Error[] Errors;

        // <remarks/>
        public string Response;

        // <remarks/>
        [XmlAttribute()]
        public string Version;
    }

    // <remarks/>
    [XmlRoot(IsNullable = false)]
    public class Success
    {
    }

    // <remarks/>
    [XmlRoot(IsNullable = false)]
    public class Error
    {

        // <remarks/>
        [XmlAttribute()]
        public string Type;

        // <remarks/>
        [XmlAttribute()]
        public string Code;

        // <remarks/>
        [XmlText()]
        public string Value;
    }

    // <remarks/>
    [XmlRoot(IsNullable = false)]
    public class Errors
    {

        // <remarks/>
        [XmlElement("Error")]
        public Error[] Error;
    }
}